src/pages/Login/LoginPage.tsx

```diff
@@ -0,0 +1,119 @@
+import { useDispatch, useSelector } from "react-redux";
+import { RootState } from "../../app/common/store";
+import { requestLogin, resetErrorMessage } from "./LoginPage.slice";
+import { IRequestLoginAction } from "./LoginPage.slice";
+import { Field, Form, Formik } from "formik";
+import { FormTextField } from "../../shared/FormTextField";
+import { LayoutContainer } from "../../shared/LayoutContainer";
+import { Routes } from "../../shared/Routes.constants";
+import { Redirect } from "react-router";
+import { LocalStorage } from "../../app/common/LocalStorage";
+import {
+	Alert,
+	Button,
+	Card,
+	CardActions,
+	CardContent,
+	CircularProgress,
+	Stack,
+} from "@mui/material";
+import styled from "@emotion/styled";
+
+//Refernce: https://github.com/creativesuraj/react-material-ui-login/blob/master/src/components/Login.tsx
+
+const FancyForm = styled(Form)`
+	display: inline-block;
+	flex-wrap: wrap;
+	align-items: center;
+	justify-content: center;
+	width: 50%;
+	margin: 12px auto;
+	left: 50%;
+	top: 50%;
+`;
+
+const FancyButton = styled(Button)`
+	flex-grow: 1;
+`;
+
+const FancyCard = styled(Card)`
+	margin-top: 12px;
+`;
```

Comment:
The code above has a fixed number of pixels for margin that doesn't depend on the `theme.spacing` property. Please modify this to use the theme.

Reply:
I couldn't figure out how to use the theme to determine spacing. I tried for like an hour. The emotion theme doesn't have a default spacing as far as I can tell.

Reply:
spacing function*

Reply:
the spacing function came from the makeStyles theme I think.

Reply:
I do not have makeStyles and I can still use it. I think it is a part of the theme.

Reply:
Should be ready to resolve.